h/feeds/render.py

```diff
@@ -1,9 +1,20 @@
+import functools
+
 from pyramid import renderers
 
+from h.features import flag_enabled
 from h.feeds import atom
 from h.feeds import rss
 
 
+def annotation_url(request, annotation):
+    """Return the HTML permalink URL for the given annotation."""
+    if flag_enabled(request, 'direct_linking'):
```

Comment:
You don't need to import `flag_enabled`. You can just ask `if request.feature('direct_linking'):`